bundles/CreateProject/Form/CreateProjectForm.php

```diff
@@ -273,6 +275,7 @@ private function getCustomerChoices()
     public function configureOptions(OptionsResolver $resolver)
     {
         $resolver->setDefaults([
+            'user_permissions' => $this->jiraService->getCurrentUserPermissions(),
```

Comment:
The form type should not call service methods here. If `user_permissions` are required, use https://symfony.com/doc/current/components/options_resolver.html#required-options. `user_permissions` should be set by the code using `CreateProjectForm`.